views/membership/index.hbs

```diff
@@ -5,14 +5,13 @@
 		<p class="mt-3 mb-3">Displaying all <b>{{membership.length}}</b> matching Memberships on record</p>
 		<a class="btn btn-light me-3" href="new/">+ Add Membership</a>
 		<a class="btn btn-success" href="import">Import Membership</a>
-
-		<div class="row mb-3 row-cols-1 row-cols-md-2 d-none">
-			<div class="col mb-3">
+		<div class="row row-cols-1 row-cols-md-3">
+			<div class="col my-3 offset-4">
 				<div class="form-group text-start">
 					<label class="form-label" for="year">Season</label>
-					<select id="year" class="form-control">
+					<select id="year" class="form-control" OnChange="location.href=year.options[selectedIndex].value">
```

Comment:
Few things on this line:

JavaScript events in attributes are sketchy at best (I'll allow it) but I actually think it looks better all lowercase

Secondly, `year` is technically undefined here - it may be nicer to use `this.options` instead of `year.options`